src/mavsdk/core/CMakeLists.txt

```diff
@@ -71,6 +71,11 @@ set_target_properties(mavsdk PROPERTIES
     VERSION ${MAVSDK_VERSION_STRING}
     SOVERSION ${MAVSDK_SOVERSION_STRING}
 )
+if (WIN32)
```

Comment:
Can we make it `if(MSVC)` instead? That's what's used everywhere else in our cmake files.